recipes/transaction/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "transaction" %}
+{% set version = "2.1.2" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "b9bc365e7dba3877e0f6fdee32aa029b8c0c1eb4fe227f524bffd5fc46064bd5" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  preserve_egg_dir: True
```

Comment:
Does `preserve_egg_dir` have to be set to true? The conda meta.yaml documentation is vague:

https://conda.io/docs/building/meta-yaml.html#preserve-python-egg-directory

so this is an honest question that I'm not sure the answer of.